src/primitives/symmetric/encryption/des/mod.rs

```diff
@@ -0,0 +1,256 @@
+//! Contains implementation of DES encryption
+#[doc = include_str!("./README.md")]
```

Comment:
This is a really nice touch. We should add this everywhere

Reply:
added to other modules as well